Testscripts/Linux/dpdkSetup.sh

```diff
@@ -83,7 +83,23 @@ function install_dpdk () {
 		dpdkSrcTar="${dpdkSrcLink##*/}"
 		dpdk_version=$(echo "$dpdkSrcTar" | grep -Po "(\d+\.)+\d+")
 		LogMsg "Installing DPDK from source file $dpdkSrcTar"
-		ssh "${1}" "wget $dpdkSrcLink -P /tmp"
+		retries=0
+		max_retries=3
+		while [[ $retries -lt $max_retries ]];
+		do
+			ssh "${1}" "wget --tries 3 --retry-connrefused '${dpdkSrcLink}' -P /tmp"
+			if [ $? = 0 ]; then
+				break
+			else:
+				retries=$(($retries+1))
+				LogErr "Could not download ${DPDK_LINK}. Retrying..."
```

Comment:
I don't prefer to add 'sleep' but shouldn't it wait for 1 or 2 seconds at gateway error?

Reply:
I can add the sleep too.
wget already retries 3 times using a default timeout, but I will add also the main retry sleep of 3 seconds (this scenario should almost never happen, but when it happens we should timeout for a longer period than 1 second).